diff --git a/dist/shared/nuxi.BVSKcfmX.mjs b/dist/shared/nuxi.BVSKcfmX.mjs
index 700741eeced6516694a6355d215f28d6011ccdaf..08ecc191e7bcdd71a37f022a4ac2d75f5daae95a 100644
--- a/dist/shared/nuxi.BVSKcfmX.mjs
+++ b/dist/shared/nuxi.BVSKcfmX.mjs
@@ -2166,11 +2166,13 @@ const main = defineCommand({
       const cwd = resolve(ctx.args.cwd);
       try {
         const { x } = await import('../chunks/main.mjs');
+        console.log(ctx, ctx.rawArgs)
         await x(`nuxt-${ctx.args.command}`, ctx.rawArgs.slice(1), {
           nodeOptions: { stdio: "inherit", cwd },
           throwOnError: true
         });
       } catch (err) {
+        console.log(err, err.toString())
         if (err instanceof Error && "code" in err && err.code === "ENOENT") {
           return;
         }
